app/models/browse_filter.rb

```diff
@@ -3,16 +3,15 @@
 # BrowseFilter lets the browse page filter contributions by parameters, delegating much of the actual filtering
 # to the classes listed in BrowseFilter::FILTER_CLASSES
 class BrowseFilter
-  FILTERS = {
-    'Category' => CategoryFilter,
-    'ServiceArea' => ServiceAreaFilter,
-    'ContactMethod' => ContactMethodFilter,
-    'ContributionType' => ContributionTypeFilter
-    # 'UrgencyLevel' => UrgencyLevelFilter
-  }.freeze
-  FILTER_CLASSES = FILTERS.values.freeze
-  ALLOWED_PARAMS = FILTERS.keys.each_with_object({}) do |key, hash|
-    hash[key] = {}
+  FILTER_CLASSES = [
+    CategoryFilter,
+    ServiceAreaFilter,
+    ContactMethodFilter,
+    ContributionTypeFilter,
+    UrgencyFilter
+  ].freeze
+  ALLOWED_PARAMS = FILTER_CLASSES.each_with_object({}) do |filter, hash|
+    hash.merge! filter::ALLOWED_PARAMS
```

Comment:
As i look at this, i'm wondering if we even need `ALLOWED_PARAMS` anymore? Since each filter will extract only the params it needs, maybe `ContributionsController` doesn't need to invoke strong_params at all? Perhaps that addresses some of the duplication you mentioned?